app/controllers/communities_controller.rb

```diff
@@ -13,6 +13,7 @@ def index
     @communities = Community.where('name ilike ?', "%#{@search_text}%") if search_params.present?
 
     redirect_to community_path(@communities.first) if @communities.count == 1
+    @communities = @communities.order(:name).page(params[:page]).per(24)
```

Comment:
I'm having difficulties adding in the default to the model, not sure if it's a rails 5 error or something I'm doing wrong.